src/test/java/dev/codesupport/web/domain/validation/MultiViolationConstraintValidator_ViolationImplTest.java

```diff
@@ -0,0 +1,168 @@
+package dev.codesupport.web.domain.validation;
+
+import dev.codesupport.web.domain.validation.MultiViolationConstraintValidator.ViolationImpl;
+import org.junit.Test;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import javax.validation.ConstraintValidatorContext;
+
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class MultiViolationConstraintValidator_ViolationImplTest {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class name 'MultiViolationConstraintValidator_ViolationImplTest' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/gh/codesupport/website-backend/pullRequest?prid=5265305)